### ISSUE TITLE
[BUG]: The project doesn't build on macOS

### ISSUE THREAD

**Issue by samueleresca** [dotnet/spark#794]
**Describe the bug**
The project returns the following error when build on MacOSX:
```
packages/microsoft.build.tasks.git/1.0.0-beta2-18618-05/build/Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018: The "Microsoft.Build.Tasks.Git.LocateRepository" task failed unexpectedly. [/Projects/spark/src/csharp/Microsoft.Spark/Microsoft.Spark.csproj]
packages/microsoft.build.tasks.git/1.0.0-beta2-18618-05/build/Microsoft.Build.Tasks.Git.targets(20,5): error MSB4018: System.TypeInitializationException: The type initializer for 'Microsoft.Build.Tasks.Git.TaskImplementation' threw an exception. [/Projects/spark/src/csharp/Microsoft.Spark/Microsoft.Spark.csproj]
```

**To Reproduce**

1. `git clone https://github.com/dotnet/spark`
2. `cd spark/src/csharp`
3. `dotnet build`

**Desktop (please complete the following information):**
- OS: macOS
- .NET Core 3.1.404

**Additional context**
Probably caused by the addition of arcade or SourceLink.

**Comment by imback82** [comment]
I cannot reproduce this with a fresh installation of dotnet 3.1.404.

**Comment by samueleresca** [comment]
Solved by reinstalling `dotnet 3.1.404` and re-cloning the repo

Thanks
